### ISSUE TITLE
API typos, outdated info, broken links, etc (ongoing)

### ISSUE THREAD

**Issue by lilyminium** [MDAnalysis/mdanalysis#2401]
# core

## topologyobjects 

`Bond.__eq__` does not check bond order:
https://github.com/MDAnalysis/mdanalysis/blob/e1101089ffc7aa58be654f7d0a82840a9d5a7175/package/MDAnalysis/core/topologyobjects.py#L163-L167

https://github.com/MDAnalysis/mdanalysis/blob/e1101089ffc7aa58be654f7d0a82840a9d5a7175/package/MDAnalysis/core/topologyobjects.py#L135-L140

# topology

All the "Reads the following Attributes" and "Guesses the following Attributes" should be double-checked.

## `__init__`

Table is out-of-date, incomplete or incorrect information, does not include ITPParser.

### PSFParser

`.. _PSF: http://www.charmm.org/documentation/c35b1/struct.html` link broken.

### MinimalParser

Ends in the middle of a sentence:
https://github.com/MDAnalysis/mdanalysis/blob/e1101089ffc7aa58be654f7d0a82840a9d5a7175/package/MDAnalysis/topology/MinimalParser.py#L58

**Comment by richardjgowers** [comment]
Yep.  So something pandas does is to mush around doc strings with decorators, so we could even do that with `topology.__init__` so that when Topology Parsers are defined, they append to that table, so they're automagically done.  Or just write some more rst if you don't want to do it programmatically.

**Comment by lilyminium** [comment]
The recently-merged #2390 was created to make a script that [auto-docs parsers and the attributes associated](https://lilyminium.github.io/UserGuide/formats/index.html#id2). It's currently in the UserGuide but I could move it over to the main repo?

**Comment by orbeckst** [comment]
@lilyminium does the UG have a list of all pre-defined topology attributes from [core.topologyattrs](https://www.mdanalysis.org/docs/documentation_pages/core/topologyattrs.html)? Would be useful for `Universe.empty()` trickery (as I found out when I wasted a few minutes answering a 4-year old Stackoverflow question [How to instantiate Atoms using MDAnalysis?](https://stackoverflow.com/questions/31582128/how-to-instantiate-atoms-using-mdanalysis)).

**Comment by lilyminium** [comment]
This example in `rms.RMSF` doesn't work, as `align.AlignTraj` requires a reference. Also, could use a trajectory that doesn't need to be made whole.

```python
import MDAnalysis as mda
from MDAnalysis.analysis import align

from MDAnalysis.tests.datafiles import TPR, XTC

u = mda.Universe(TPR, XTC, in_memory=True)
protein = u.select_atoms("protein")

# 1) need a step to center and make whole: this trajectory
#    contains the protein being split across periodic boundaries
#
# TODO

# 2) fit to the initial frame to get a better average structure
#    (the trajectory is changed in memory)
prealigner = align.AlignTraj(u, select="protein and name CA", in_memory=True).run()

# 3) reference = average structure
reference_coordinates = u.trajectory.timeseries(asel=protein).mean(axis=1)
# make a reference structure (need to reshape into a 1-frame "trajectory")
reference = mda.Merge(protein).load_new(
            reference_coordinates[:, None, :], order="afc")
```

**Comment by jbarnoud** [comment]
> Also, could use a trajectory that doesn't need to be made whole.

Not dealing with PBC is a frequent misuse of the analysis tools. Showing it has to be done seems a good idea to me. Our examples will be used more or less verbatim in papers.
